freqtrade/freqai/data_kitchen.py

```diff
@@ -320,8 +327,14 @@ def normalize_data(self, data_dictionary: Dict) -> Dict[Any, Any]:
                     - 1
                 )
 
-            self.data[f"{item}_max"] = train_labels_max  # .to_dict()
-            self.data[f"{item}_min"] = train_labels_min  # .to_dict()
+            if not [col for col in df_train_features.columns if col.startswith('PC')]:
```

Comment:
Same as comment above, I dont think we need this check since `item` is unique already wrt principal components or normal features.